pkgs/os-specific/linux/veikk/gui/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, gnumake, qmake }:
+
+mkDerivation rec {
+  name = "veikk-linux-driver-gui";
+  version = "v2.0";
+
+  src = fetchFromGitHub {
+    owner = "jlam55555";
+    repo = name;
+    rev = version;
+    sha256 = "02g1q79kwjlzg95w38a1d7nxvcry8xcsvhax2js4c7xqvzhkki5j";
+  };
+
+  nativeBuildInputs = [ qmake gnumake ];
```

```suggestion
  nativeBuildInputs = [ qmake ];
```
already part of stdenv.